packages/components/inputs/radio-input/src/radio-input.story.js

```diff
@@ -101,6 +101,7 @@ storiesOf('Components|Inputs', module)
                 radioOption3
               )}
               isHovered={boolean('isHovered #3', false, radioOption3)}
+              additionalContent="Lorem ipsum dolor sit amet, consetetur sadipscing elitr"
```

Comment:
I think it would be great to also add a new use case to the [visualroute tests](https://github.com/commercetools/ui-kit/blob/main/packages/components/inputs/radio-input/src/radio-input.visualroute.js) so we can have a regression test for this new type of visualization.